python/src/energuide/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class InvalidGroupSizeException(Exception):
+    pass
+
+
+class InvalidInputDataException(Exception):
+    pass
+
+
+class InvalidEmbeddedDataTypeException(Exception):
```

Comment:
We should probably have a custom base class for all our custom exceptions, so at a root level we can just catch the one base class.